railties/lib/rails/generators/rails/app/templates/config/cable.yml.tt

```diff
@@ -6,5 +6,5 @@ test:
 
 production:
   adapter: redis
-  url: <%%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
+  url: <%%= ENV.fetch("CABLE_REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
That will potentially break all running apps in production, without any notice? Because, `REDIS_URL` is so common used.

Reply:
We are going to just still with `REDIS_URL`. However, this is just a template for generating new apps / installing action cable, so changes here should not impact existing apps already using action cable.

Reply:
You are right, I was thinking about upgrades. But anyway, like the fact, you are reverted it to `REDIS_URL` 👍🏼 .